src/demo/App.js

```diff
@@ -38,6 +39,35 @@ const demos = {
             ]
         }
     },
+    "H3K27ac Demo (1 View, Center Track, Rows Aggregated)": {
+        viewConfig: hgDemoViewConfig1b,
+        options: {
+            rowInfoAttributes: [
+                {field: "Hierarchical Clustering (Average)", type: "tree", position: "left"},
+                {field: ["qc_frip", "qc_fastqc"], type: "quantitative", position: "left", aggFunction: "mean"},
+                {field: "qc_frip", type: "quantitative", position: "left", aggFunction: "mean"},
+                {field: "qc_fastqc", type: "quantitative", position: "left", aggFunction: "mean"},
+                {field: "Metadata URL", type: "url", position: "left", title: "cid", aggFunction: "max"},
+                {field: "Hierarchical Clustering (Ward)", type: "tree", position: "right"},
+                {field: "Cell Type", type: "nominal", position: "right", aggFunction: "sum"},
+                {field: "Tissue Type", type: "nominal", position: "right", aggFunction: "sum"},
+                {field: "Species", type: "nominal", position: "right", aggFunction: "sum"}
+            ],
+            rowAggregate: [
+                {field: "Cell Type", type: "nominal", oneOf: ["Fibroblast", "Epithelium"]},
```

Comment:
This syntax seems good for aggregating by a small number cell types that I know in advance or select in an interface, but what if I do not know the cell type names in advance, and instead wanted to aggregate all samples with the same cell type value? That is the use case that I have been thinking about, which I think will be useful when the multivec file is very large.

Maybe this can be the behavior when the `oneOf` property is missing / undefined?

Reply:
Good point. I added a few lines to enable aggregating entire rows by categories in the certain field when `oneOf` is not defined: https://github.com/hms-dbmi/cistrome-explorer/blob/a9994547f3dc5d67f63134271565e8bf0166a2fb/src/utils/select-rows.js#L171